demo/apiExamples.md

```diff
@@ -8,10 +8,10 @@
 |---------------|---------------|-----------|-------------|--------------------------------------------------|
 | [centralDate](#centralDate) | `centralDate` | `Object`  |             | The date that determines the currently visible month. |
 | [disabled](#disabled)    | `disabled`    | `Boolean` |             | If set, disables the datepicker.                 |
-| [error](#error)       | `error`       | `Boolean` | false       | Sets a persistent error state (e.g. an error state returned from the server). |

```

Comment:
Ok, we are removing this, but we should say something about the fact that `validity` is talking to `input` and then addressing the error state. 

I understand that at this point the user does not have a way to inject a custom error message, so when that comes about, how do you see supporting that?

Reply:
The example for this is still in the docs.